native/textern.py

```diff
@@ -78,6 +102,7 @@ def main():
         loop.add_reader(sys.stdin.buffer, handle_stdin, tmp_mgr)
         loop.add_reader(ino.fd, handle_inotify_event, ino, tmp_mgr)
         loop.run_forever()
+        tmp_mgr.kill_editors()
```

Comment:
Maybe better to move this after the next line?

Reply:
> Maybe better to move this after the next line?

That's possible, no problem.